os/booting-on-ecs.md

```diff
@@ -21,21 +21,37 @@ coreos:
      runtime: true
      content: |
        [Unit]
-       Description=Amazon ECS Agent
-       After=docker.service
-       Requires=docker.service
-       Requires=network-online.target
-       After=network-online.target
-       
+       Description=AWS ECS Agent (https://github.com/aws/amazon-ecs-agent)
```

Comment:
Please remove the URL from the description so that it doesn't clutter up the logs.